api/handlers.go

```diff
@@ -101,11 +109,16 @@ func validateTransactionFilter(filter *idb.TransactionFilter) error {
 // Returns 200 if healthy.
 // (GET /health)
 func (si *ServerImplementation) MakeHealthCheck(ctx echo.Context) error {
+	var err error
 	var errors []string
+	var health idb.Health
 
-	health, err := si.db.Health()
+	err = util.CallWithTimeout(ctx.Request().Context(), si.log, si.timeout, func(ctx context.Context) error {
```

Comment:
I considered it but would prefer to keep it separate. I think it's good to make sure all the inputs are visible and obvious, mainly since the function is doing concurrency things.

Reply:
Sure, I'll make them private also.